src/styles/_catalog.scss

```diff
@@ -1,22 +1,28 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
+  display: grid;
+  gap: 48px;
+  justify-content: center;
   margin: 0 auto;
+  grid-template-columns: repeat(1, $cardWidth);
```

Comment:
I think you don't need this value here, they will always set automatically 1 element in breakpoint below 488px